ui/src/ducks/app/volumes.js

```diff
@@ -212,6 +212,7 @@ export function* createVolumes({ payload }) {
     kind: 'Volume',
     metadata: {
       name: newVolume.name,
+      labels: newVolume.labels,
```

Comment:
Oh no....... I missed this, labels were not supposed to go there, but in `spec.template.metadata.labels` :(

Reply:
Ha indeed, otherwise we're labeling the volume itself and not the underlying PV.